concepticondata/conceptlists/Valenzuela-2023-49.tsv

```diff
@@ -0,0 +1,50 @@
+ID	NUMBER	NUMBERINSOURCE	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Valenzuela-2023-49-1	1	1	Tree	906	TREE
```

Comment:
We should have added the "Table" information in "concepts", not in the raw data. Could you add the "Table" information to this file?

Reply:
I'm not sure about the "Number in Source" info. I haven't seen that before in our lists - is this relevant?

Reply:
If it's relevant information, the header should be changed to `NUMBER_IN_SOURCE`.

Reply:
Yes, it is. We changed the numbering of the original source for the dataset that we were building. So, we kept both numberings and I just changed the header to `NUMBER_IN_TABLE` for the last update.